### ISSUE TITLE
Look into pickle deprecation warnings

### ISSUE THREAD

**Issue by abingham** [sixty-north/cosmic-ray#118]
When I run worker tasks right now I see:
```
[2015-12-26 15:17:57,622: WARNING/MainProcess] /Users/sixtynorth/.virtualenvs/with_fixture/lib/python3.5/site-packages/celery-3.1.19-py3.5.egg/celery/apps/worker.py:161: CDeprecationWarning: 
Starting from version 3.2 Celery will refuse to accept pickle by default.

The pickle serializer is a security concern as it may give attackers
the ability to execute any command.  It's important to secure
your broker from unauthorized access when using pickle, so we think
that enabling pickle should require a deliberate action and not be
the default choice.

If you depend on pickle then you should set a setting to disable this
warning and to be sure that everything will continue working
when you upgrade to Celery 3.2::

    CELERY_ACCEPT_CONTENT = ['pickle', 'json', 'msgpack', 'yaml']

You must only enable the serializers that you will actually use.


  warnings.warn(CDeprecationWarning(W_PICKLE_DEPRECATED))
```

**Comment by abingham** [comment]
[Here](http://docs.celeryproject.org/en/latest/userguide/application.html#configuration) are instructions for setting configuration values.

**Comment by abingham** [comment]
Fixed in 0d5e09a873c682e82564b3f3a8d9c76d087e6536.
